lib/cloudflare/connection.rb

```diff
@@ -29,27 +29,27 @@
 module Cloudflare
 	DEFAULT_URL = "https://api.cloudflare.com/client/v4/"
 	TIMEOUT = 10 # Default is 5 seconds
-	
+
```

Comment:
looks like you could restore the master version of this file to minimize the diff